### ISSUE TITLE
[​​​​Fanboy’s Social Blocking List] Breaks “Share Link” functionality on any recent NextCloud instance

### ISSUE THREAD

**Issue by alexander255** [ryanbr/fanboy-adblock#1225]
### List the website(s) you're having issues:

 * `https://storage.ninetailed.ninja/` (our instance that is affected by this)
 * Any other company/individual/organization using NextCloud (nextcloud.com) – at least several thousand instances with upwards of 50 000 users
 * Example of random person on the internet affected by this: `https://www.reddit.com/r/NextCloud/comments/brkmlm/share_link_not_clickable_on_files_and_folders/`

### What happens?

The option to “Share Link” (ie: share a file by generating a unique publi link), appears to be broken as the the “+”-button next to the text is hidden by the `##.share-menu` entry rule at https://github.com/ryanbr/fanboy-adblock/blob/master/fanboy-addon/fanboy-addon-elements.txt#L1465

(If you insist on it I can create dummy profile for to verify.)

### List Subscriptions you're using:

I've already narrowed this down to “Fanboy's Social Blocking List” as shipped with uBlockOrigin. The issue is reproducible even if only that list is enabled.

### Your settings

- OS/version: Debian 10
- Browser/version: Firefox Developer Edition ~70
- Adblock Extension/version: uBlock Origin v1.23.0

### Other details:

See the “What happens” section above.

**Comment by krystian3w** [comment]
https://github.com/AdguardTeam/AdguardFilters/issues/51643#issuecomment-599787060

**Comment by pktiuk** [comment]
It still happens on my instance.

**Comment by krystian3w** [comment]
I don't think so, it requires abandoning generics or rebuilding them so that they don't fit into most private next cloud sites.

You could try adding this to your filters:

```adb
@@||here-your-private-domain.foo^$generichide
```

**Comment by pktiuk** [comment]
> I don't think so, it requires abandoning generics or rebuilding them so that they don't fit into most private next cloud sites.

That's a shame

> You could try adding this to your filters:

Thanks, I will try
